seatunnel-connectors-v2/connector-socket/src/main/java/org/apache/seatunnel/connectors/seatunnel/socket/config/SinkConfig.java

```diff
@@ -25,13 +25,14 @@
 
 @Data
 public class SinkConfig implements Serializable {
-    public static final  String HOST = "host";
-    public static final  String PORT = "port";
-    private static final  String MAX_RETRIES = "max_retries";
-    private static final  int DEFAULT_MAX_RETRIES = 3;
+    public static final String HOST = "host";
+    public static final String PORT = "port";
+    private static final String MAX_RETRIES = "max_retries";
+    private static final int DEFAULT_MAX_RETRIES = 3;
     private String host;
-    private Integer port;
+    private int port;
     private Integer maxNumRetries = DEFAULT_MAX_RETRIES;
```

Comment:
The same as `port`

Reply:
Thank you very much for your advice. I have fixed the problems mentioned above